### PR TITLE
fix: update anchor styles and add destination to links

### PR DESCRIPTION
**Pull request by khamudom**
# Description

This change has a destination page for the links, that are available. The styles on the links, on both full-page, in the header and footer, and inside the nav flyout, are updated, overriding the accent color and size. Updated the logo without the 'FAST' label as part of the icon.

image
image
image

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-website/src/public/index.html [hunk 1]
```diff
@@ -98,19 +99,19 @@ <h2 class="section-heading">Join the community</h2>
...
-              <li><fast-anchor appearance="lightweight" href="#">Privacy & Cookies</fast-anchor></li>
-              <li><fast-anchor appearance="lightweight" href="#">Terms of Use</fast-anchor></li>
-              <li><fast-anchor appearance="lightweight" href="#">&copy; 2020 Microsoft</fast-anchor></li>
-            </ul>
-          </site-navigation>
-          <fast-anchor class="logo" appearance="lightweight" href="#">
-            <%= require("svg/icon-brand.svg") %>
-          </fast-anchor>
-        </footer>
+          <div class="footer">
```
Comment (radium-v) [inline sites/fast-website/src/public/index.html:102]: This can still be a `<footer>` element

File: sites/fast-website/src/public/index.html [hunk 2]
```diff
@@ -98,19 +99,19 @@ <h2 class="section-heading">Join the community</h2>
...
+            <site-navigation>
+              <ul>
+                <li><fast-anchor appearance="lightweight" href="https://github.com/microsoft/fast-dna/blob/master/LICENSE">License</fast-anchor></li>
+                <li><fast-anchor appearance="lightweight" href="https://privacy.microsoft.com/en-US/privacystatement">Privacy & Cookies</fast-anchor></li>
+                <li><fast-anchor appearance="lightweight" href="https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx">Terms of Use</fast-anchor></li>
+                <li><p>&copy; 2020 Microsoft</p></li>
+              </ul>
+            </site-navigation>
+            <fast-anchor class="logo" appearance="lightweight" href="#">
```
Comment (radium-v) [inline sites/fast-website/src/public/index.html:111]: hash link

Comment (khamudom) [reply sites/fast-website/src/public/index.html:111]: What url should I set for the logo link?

Comment (radium-v) [reply sites/fast-website/src/public/index.html:111]: Probably either `/` or `https://fast.design`

Comment (khamudom) [reply sites/fast-website/src/public/index.html:111]: thanks, using `/`

File: sites/fast-website/src/public/index.html (hunk 1 above)
Comment (khamudom) [reply sites/fast-website/src/public/index.html:102]: thanks, updated

**Pull request merged**